src/main.rs

```diff
@@ -153,6 +156,9 @@ fn main() {
     let mut runner = Runner::new(args.is_present("restart"), args.is_present("clear"));
 
     loop {
+        if args.is_present("run-initially") {
```

Comment:
I think you want this to be outside the loop so it doesn't run twice with every change.